Routing/RoutingService.swift

```diff
@@ -39,10 +53,58 @@ public enum RoutingError: ErrorType {
     case BadRequest(String)
     case ServerError(String)
     case UnknownError
+
+    public func rawValue() -> Int {
```

Comment:
This is a bit unpleasant, but a swift enum with parameters can't be `RawRepresentable` as well, so this has to exist as far as I can tell.